### ISSUE TITLE
Settings crash on SDK < 19

### ISSUE THREAD

**Issue by yuriykulikov** [yuriykulikov/AlarmClock#346]
## Trace
```
APP_VERSION_CODE=30700
25-02 23:16:36 DBG [         AlarmSetter] SDK is 19 
...
25-02 23:16:40 DBG [         BugReporter] Uncaught exception java.lang.RuntimeException: Unable to start activity ComponentInfo{com.better.alarm/com.better.alarm.presenter.SettingsActivity}: java.lang.IllegalStateException: You need to use a Theme.AppCompat theme (or descendant) with this activity. Unable to start activity ComponentInfo{com.better.alarm/com.better.alarm.presenter.SettingsActivity}: java.lang.IllegalStateException: You need to use a Theme.AppCompat theme (or descendant) with this activity.
STACK_TRACE=java.lang.RuntimeException: Unable to start activity ComponentInfo{com.better.alarm/com.better.alarm.presenter.SettingsActivity}: java.lang.IllegalStateException: You need to use a Theme.AppCompat theme (or descendant) with this activity.
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2235)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2285)
	at android.app.ActivityThread.access$800(ActivityThread.java:138)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1236)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:149)
	at android.app.ActivityThread.main(ActivityThread.java:5061)
	at java.lang.reflect.Method.invokeNative(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:515)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:793)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:609)
	at dalvik.system.NativeStart.main(Native Method)
Caused by: java.lang.IllegalStateException: You need to use a Theme.AppCompat theme (or descendant) with this activity.
	at androidx.appcompat.app.AppCompatDelegateImpl.createSubDecor(AppCompatDelegateImpl.java:696)
	at androidx.appcompat.app.AppCompatDelegateImpl.ensureSubDecor(AppCompatDelegateImpl.java:659)
	at androidx.appcompat.app.AppCompatDelegateImpl.setContentView(AppCompatDelegateImpl.java:552)
	at androidx.appcompat.app.AppCompatActivity.setContentView(AppCompatActivity.java:161)
	at com.better.alarm.presenter.SettingsActivity.onCreate(SettingsActivity.java:42)
	at android.app.Activity.performCreate(Activity.java:5231)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1087)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2199)
	... 11 more
```

**Comment by yuriykulikov** [comment]
```
IS_SILENT=false
APP_VERSION_CODE=30811
PHONE_MODEL=ORINOQUIA Auyantepui+Y221-U03
ANDROID_VERSION=4.4.2
CUSTOM_DATA=STARTUP_LOG=
18-08 01:46:30 DBG [         AlarmSetter] SDK is 19 
18-08 01:46:30 DBG [              Alarms] Starting alarms 
18-08 01:46:30 DBG [           AlarmCore] [Alarm 1] entering [RootState, DisabledState] 
18-08 01:46:30 DBG [           AlarmCore] [Alarm 2] entering [RootState, DisabledState] 
18-08 01:46:30 DBG [              Alarms] 1 [ ] 8:30 mtwtf__  
18-08 01:46:30 DBG [              Alarms] 2 [ ] 9:0 _____ss  
18-08 01:46:30 DBG [              Alarms] ## Next: Optional(of=null) 
18-08 01:46:30 DBG [  AlarmsListActivity] Created com.better.alarm.presenter.AlarmsListActivity@421027b8 with EditedAlarm(isNew=false, id=-1, value=Optional(of=null), holder=Optional(of=null)) 
18-08 01:46:30 DBG [  AlarmsListActivity] transition from: null to show list, edited: EditedAlarm(isNew=false, id=-1, value=Optional(of=null), holder=Optional(of=null)) 
18-08 01:46:30 DBG [  AlarmsListFragment] onCreateView AlarmsListFragment{41ff1f28} (4625e625-c6ca-4daa-a4b0-7783bd2b106b) id=0x7f0900c4} 
18-08 01:46:31 DBG [         BugReporter] Uncaught exception android.view.InflateException: Binary XML file line #101: Error inflating class ImageView Binary XML file line #101: Error inflating class ImageView
STACK_TRACE=android.view.InflateException: Binary XML file line #101: Error inflating class ImageView
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:713)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:755)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:758)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:758)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:492)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:397)
	at com.better.alarm.presenter.AlarmsListFragment$AlarmListAdapter.recycleView(AlarmsListFragment.kt:78)
	at com.better.alarm.presenter.AlarmsListFragment$AlarmListAdapter.getView(AlarmsListFragment.kt:90)
	at android.widget.AbsListView.obtainView(AbsListView.java:2338)
	at android.widget.ListView.makeAndAddView(ListView.java:1812)
	at android.widget.ListView.fillDown(ListView.java:698)
	at android.widget.ListView.fillFromTop(ListView.java:759)
	at android.widget.ListView.layoutChildren(ListView.java:1631)
	at android.widget.AbsListView.onLayout(AbsListView.java:2149)
	at android.view.View.layout(View.java:15154)
	at android.view.ViewGroup.layout(ViewGroup.java:4872)
	at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1888)
	at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1742)
	at android.widget.LinearLayout.onLayout(LinearLayout.java:1651)
	at android.view.View.layout(View.java:15154)
	at android.view.ViewGroup.layout(ViewGroup.java:4872)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:515)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:450)
	at android.view.View.layout(View.java:15154)
	at android.view.ViewGroup.layout(ViewGroup.java:4872)
	at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1888)
	at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1742)
	at android.widget.LinearLayout.onLayout(LinearLayout.java:1651)
	at android.view.View.layout(View.java:15154)
	at android.view.ViewGroup.layout(ViewGroup.java:4872)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:515)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:450)
	at android.view.View.layout(View.java:15154)
	at android.view.ViewGroup.layout(ViewGroup.java:4872)
	at androidx.appcompat.widget.ActionBarOverlayLayout.onLayout(ActionBarOverlayLayout.java:446)
	at android.view.View.layout(View.java:15154)
	at android.view.ViewGroup.layout(ViewGroup.java:4872)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:515)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:450)
	at android.view.View.layout(View.java:15154)
	at android.view.ViewGroup.layout(ViewGroup.java:4872)
	at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1888)
	at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1742)
	at android.widget.LinearLayout.onLayout(LinearLayout.java:1651)
	at android.view.View.layout(View.java:15154)
	at android.view.ViewGroup.layout(ViewGroup.java:4872)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:515)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:450)
	at android.view.View.layout(View.java:15154)
	at android.view.ViewGroup.layout(ViewGroup.java:4872)
	at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:2347)
	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2053)
	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1216)
	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:6287)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:788)
	at android.view.Choreographer.doCallbacks(Choreographer.java:591)
	at android.view.Choreographer.doFrame(Choreographer.java:560)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:774)
	at android.os.Handler.handleCallback(Handler.java:808)
	at android.os.Handler.dispatchMessage(Handler.java:103)
	at android.os.Looper.loop(Looper.java:193)
	at android.app.ActivityThread.main(ActivityThread.java:5299)
	at java.lang.reflect.Method.invokeNative(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:515)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:822)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:638)
	at dalvik.system.NativeStart.main(Native Method)
Caused by: java.lang.NumberFormatException: Invalid int: "res/color/abc_primary_text_material_dark.xml"
	at java.lang.Integer.invalidInt(Integer.java:137)
	at java.lang.Integer.parse(Integer.java:374)
	at java.lang.Integer.parseInt(Integer.java:365)
	at com.android.internal.util.XmlUtils.convertValueToInt(XmlUtils.java:122)
	at android.content.res.TypedArray.getInt(TypedArray.java:255)
	at android.widget.ImageView.<init>(ImageView.java:155)
	at androidx.appcompat.widget.AppCompatImageView.<init>(AppCompatImageView.java:72)
	at androidx.appcompat.widget.AppCompatImageView.<init>(AppCompatImageView.java:68)
	at androidx.appcompat.app.AppCompatViewInflater.createImageView(AppCompatViewInflater.java:187)
	at androidx.appcompat.app.AppCompatViewInflater.createView(AppCompatViewInflater.java:107)
	at androidx.appcompat.app.AppCompatDelegateImpl.createView(AppCompatDelegateImpl.java:1407)
	at androidx.appcompat.app.AppCompatDelegateImpl.onCreateView(AppCompatDelegateImpl.java:1457)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:684)
	... 66 more
```

**Comment by yuriykulikov** [comment]
Fixed by 7d5a43e0468e624be22397f35fb142b079c6da4d
